Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -305,7 +306,8 @@ public override void Render(IRenderContext rc)
                     this.MarkerStroke,
                     this.MarkerStrokeThickness,
                     this.EdgeRenderingMode,
-                    1);
+                    ActualDashArray,
```

Comment:
Should be `this.ActualDashArray`

Reply:
Should actually be `this.ActualMarkerDashArray` 🙈